.github/workflows/e2e.yml

```diff
@@ -45,6 +54,9 @@ jobs:
       - name: Start http server
         run: yarn serve:build &
 
+      - name: Set branch and commit as browserstack info
```

Comment:
This can also be done in the BrowserStack env action by setting
`build-name`

Reply:
Several tests showed that this GH-action field is not working as expected.

Theoretically there are two pre-defined constants "BUILD_INFO" and "REPO_NAME" which could be used for the action properties _build-name_ and _repo-name_. While the action outputs the correct values (BUILD_INFO looks pretty nice btw), they aren't used in the web dashboard. Instead the values of the browserstack.json were used. I removed them but that didn't help either.
The dashboard just says "cypress build" and "untitled cypress project".
I then thought that maybe the constants are a problem and used my manual commit and branch extraction. Result: Cypress build, untitles cypress project.

It seems that the GitHub action for setting up the test environment is just not properly working. It always shows the correct build-name and repo-name in its log but doesn't transmit them to browserstack.
![image](https://user-images.githubusercontent.com/52606896/104173626-5ce54b80-5406-11eb-9d02-1becc4a9cd30.png)


For now I'm reverting to jq-ing the browserstack.json - this at least is working.

Reply:
sad.. but okay.